node/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 - for information on the respective copyright owner
+// see the NOTICE file and/or the repository at
+// https://github.com/direct-state-transfer/dst-go
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package node implements the an node that can run multiple instances of
```

Comment:
```suggestion
// Package node implements a node that can run multiple instances of
```
i do not find the *node* that it speaks of btw.

Reply:
The node itself will be implemented in the later tasks. The scope of this pull request is initialize the client, identify the necessary components and provide an initial structure to the package. 

Currently, only user initialization functionality is implemented in this package.